### DISCUSSION TITLE
Duda con temario/4.-acomodoNavegacion.md

### DISCUSSION THREAD

**Discussion by FMartinRG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#390] [Q&A]
Buenas tardes compañeros, una duda intento encontrar en los videos de semana 2 donde se explica el código de:

temario/4.-acomodoNavegacion.md

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/blob/main/02%20-%20HTML/temario/4.-acomodoNavegacion.md

sin embargo no lo encuentro si alguien lo ubica y me pudiera decir les agradecería mucho ya que en los videos de semana 2 no lo encuentro. De antemano gracias!!

**Comment by doguedogue** [comment]
aquí están todos los videos del MC, procura tener abierta tu cuenta de office con tu cuenta de innovaccion

https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a

**Reply by FMartinRG** [reply]
Gracias por tomarte el tiempo de responder buena tarde

**Comment by DanielOe21** [comment]
Tengo la misma duda, parece que los 2 ultimos contenidos de HTML solo están en archivo , pero no encuentro el video explicativo :/
